manifests/router/service.pp

```diff
@@ -4,8 +4,8 @@
 class qpid::router::service {
 
   service { 'qdrouterd':
-    ensure     => running,
-    enable     => true,
+    ensure     => $qpid::router::service_ensure,
+    enable     => $qpid::router::service_enable,
```

Comment:
Is it intentional or an oversight that we don't use the same pattern here?

```suggestion
    enable     => pick($qpid::router::service_enable, $qpid::router::service_ensure),
```

If there is no reason for the different behavior here, then let's make it consistent.

Reply:
Blind addressing of direct comments :) I have updated both to the same pattern now with tests.

Reply:
Great, thanks!